src/main/java/com/ostapula/maven/sort/Person.java

```diff
@@ -0,0 +1,58 @@
+package com.ostapula.maven.sort;
+
+public class Person implements Comparable<Person>{
+
+	private final String name;
+	private final String surname;
+
+	protected Person(String name, String surname) {
```

Comment:
@Ostapula what is the purpose for making this constructor protected?

Reply:
I used to think it would be a great idea to make everything as inaccessible as possible, but later I forgot to change it, as it makes no sens.